Tests 2025 12

ue71603 opened the pull request:
* Fixed an inconsistency in some scripts
* Started to upgrade the /fix things to the new file handler

ue71603 committed:
streamlined a mistake in the testscripts

ue71603 committed:
Adapting  netex_uicoperatingperiod_correction to the new file handler

part 1

skinkie commented:
I think the same file structure:
```
def script_name(path: Path):

def main():
 # checks the input
 script_name()
```

...like the conv folder would be good for the tools as well.

ue71603 committed:
Update netex_uicoperatingperiod_correction.py

ue71603 committed:
Create __init__.py

ue71603 committed:
update ignore for xls with results

ue71603 committed:
Updates to script runnder and logging

- logging exception with stack trace implemented
- If in the script an exception occur this no longer results in a return code of 0

ue71603 committed:
new runner that executes meta_list.txt as scripts

skinkie committed:
@ue71603 please always run black pre commit
